server/src/controllers/maintenance.controller.ts

```diff
@@ -38,12 +85,81 @@ export class MaintenanceController {
     @GetLoginDetails() loginDetails: LoginDetails,
     @Res({ passthrough: true }) res: Response,
   ): Promise<void> {
-    if (dto.action === MaintenanceAction.Start) {
-      const { jwt } = await this.service.startMaintenance(auth.user.name);
+    if (dto.action !== MaintenanceAction.End) {
+      const { jwt } = await this.service.startMaintenance(dto, auth.user.name);
       return respondWithCookie(res, undefined, {
         isSecure: loginDetails.isSecure,
         values: [{ key: ImmichCookie.MaintenanceToken, value: jwt }],
       });
     }
   }
+
+  @Get('backups/list')
+  @Endpoint({
+    summary: 'List backups',
+    description: 'Get the list of the successful and failed backups',
+    history: new HistoryBuilder().added('v9.9.9').alpha('v9.9.9'),
+  })
+  @Authenticated({ permission: Permission.Maintenance, admin: true })
+  listBackups(): Promise<MaintenanceListBackupsResponseDto> {
+    return this.service.listBackups();
+  }
+
+  @Get('backups/:filename')
```

Comment:
All of these endpoints are using the terminology of "Backups", but really they are database backups and in the future we might have other types of backups. Also, getting backup information should not be restricted or tied to maintenance mode imo. Can we pull all of these endpoints out into a new controller called `DatabaseBackupController` at use the `/database-backups` path instead?